go.mod

```diff
@@ -3,34 +3,75 @@ module sigs.k8s.io/cloud-provider-azure
+	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
+	github.com/golang/mock v1.4.3 // indirect
+	github.com/google/go-cmp v0.5.0 // indirect
+	github.com/googleapis/gnostic v0.4.2 // indirect
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/imdario/mergo v0.3.9 // indirect
+	github.com/konsorten/go-windows-terminal-sequences v1.0.3 // indirect
+	github.com/mailru/easyjson v0.7.1 // indirect
+	github.com/moby/term v0.0.0-20200611042045-63b9a826fb74 // indirect
+	github.com/onsi/ginkgo v1.14.0
+	github.com/onsi/gomega v1.10.1
+	github.com/opencontainers/go-digest v1.0.0 // indirect
+	github.com/prometheus/client_golang v1.7.1
+	github.com/rubiojr/go-vhd v0.0.0-20200706122120-ccecf6c0760f // indirect
+	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.4.0
-	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.0.0
-	k8s.io/apiserver v0.0.0
-	k8s.io/client-go v0.0.0
-	k8s.io/cloud-provider v0.0.0
-	k8s.io/component-base v0.0.0
+	github.com/stretchr/testify v1.6.1
+	go.uber.org/atomic v1.6.0 // indirect
+	go.uber.org/multierr v1.5.0 // indirect
+	go.uber.org/zap v1.15.0 // indirect
+	golang.org/x/crypto v0.0.0-20200709230013-948cd5f35899 // indirect
+	golang.org/x/net v0.0.0-20200707034311-ab3426394381 // indirect
+	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d // indirect
+	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208 // indirect
+	golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae // indirect
+	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e // indirect
+	google.golang.org/appengine v1.6.6 // indirect
+	google.golang.org/genproto v0.0.0-20200711021454-869866162049 // indirect
+	google.golang.org/grpc v1.30.0 // indirect
+	gopkg.in/square/go-jose.v2 v2.5.1 // indirect
+	gopkg.in/yaml.v2 v2.3.0 // indirect
+	k8s.io/api v0.18.5
+	k8s.io/apimachinery v0.18.5
+	k8s.io/apiserver v0.18.5
+	k8s.io/client-go v1.5.1
+	k8s.io/cloud-provider v0.18.5
+	k8s.io/component-base v0.18.5
 	k8s.io/klog v1.0.0
-	k8s.io/kube-controller-manager v0.0.0
-	k8s.io/kubernetes v0.0.0-00010101000000-000000000000
-	k8s.io/legacy-cloud-providers v0.0.0
-	k8s.io/utils v0.0.0-20200324210504-a9aa75ae1b89
+	k8s.io/klog/v2 v2.3.0 // indirect
+	k8s.io/kube-controller-manager v0.18.5
+	k8s.io/kube-openapi v0.0.0-20200615155156-dffdd1682719 // indirect
+	k8s.io/kubectl v0.18.5 // indirect
+	k8s.io/kubernetes v1.18.5
+	k8s.io/legacy-cloud-providers v0.18.5
+	k8s.io/utils v0.0.0-20200619165400-6e3d28b6ed19
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.0.10 // indirect
 )
 
 replace (
 	bitbucket.org/bertimus9/systemstat => bitbucket.org/bertimus9/systemstat v0.0.0-20180207000608-0eeff89b0690
 	cloud.google.com/go => cloud.google.com/go v0.38.0
-	github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v40.2.0+incompatible
+	github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v44.0.1-0.20200710185145-8277be38f5a2+incompatible
```

Comment:
in Line6 it's `v44.0.0`, inconsistent with this line.
Pls also paste all command you used for this go mod change.

Reply:
just `go get -u ./...` after manually modify some lines, for example, link k8s.ui/cloud-provider from k/staging to external repo.

Reply:
> just `go get -u ./...` after manually modify some lines, for example, link `k8s.io/cloud-provider` from `k/staging` to external repo.

Reply:
what about: 
edit go.mod
```console
go mod tidy
go mod vendor
```

Reply:
yeah these two commands are executed after updating `go/mod`.

Reply:
usually I won't run `go get -u ./...` after go.mod change, only
```
#edit go.mod
go mod tidy
go mod vendor
```

Reply:
got it, updated.